include/NovelRT/DebugService.h

```diff
@@ -25,11 +30,13 @@ namespace NovelRT {
 
     bool getIsFpsCounterVisible() const;
     void setIsFpsCounterVisible(bool value);
-
+    void getStatistics(uint32_t framesPerSecond,uint32_t totalSeconds,uint32_t totalFrames);
```

Comment:
How come this is called getStatistics when it is a void returning method?

Reply:
I chose get because I thought it fit into the update category, but I will change it maybe "setAccumalatedStatitistics"?

Reply:
If it is setting them then it makes sense - generally the name should reflect what it does after all. :D